src/styles/utils/index.ts

```diff
@@ -1559,7 +1559,7 @@ const createStyleUtils = (theme: ThemeColors, styles: ThemeStyles) => ({
                 columnWidth = {...getWidthStyle(variables.w36), ...styles.alignItemsCenter};
                 break;
             case CONST.SEARCH_TABLE_COLUMNS.DATE:
-                columnWidth = getWidthStyle(shouldExtendDateColumn ? variables.w80 : variables.w44);
+                columnWidth = getWidthStyle(shouldExtendDateColumn ? variables.w80 : variables.w50);
```

Comment:
FWIW, 80 isn't quite big enough when we have a year present:
![CleanShot 2024-06-19 at 16 40 39@2x](https://github.com/Expensify/App/assets/2319350/57d0319e-8f5c-44e8-b246-60798971f029)

And for the other case, instead of 50, can you try using 52 or 56?